runio.js

```diff
@@ -0,0 +1,196 @@
+      cfg.line('title: Community Helpers');
+      cfg.line('---');
+      cfg.line('');
+      cfg.line('> Share your helpers at our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Community-Helpers)');
+      cfg.line('');
+      cfg.textFromFile('website/wiki/Community-Helpers.md');
+    });
+
+    writeToFile('docs/examples.md', (cfg) => {
+      cfg.line('---');
+      cfg.line('id: examples');
+      cfg.line('title: Examples');
+      cfg.line('---');
+      cfg.line('');
+      cfg.line('> Add your own examples to our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Examples)');
+      cfg.textFromFile('website/wiki/Examples.md');
+    });
+
+    writeToFile('docs/books.md', (cfg) => {
+      cfg.line('---');
+      cfg.line('id: books');
+      cfg.line('title: Books & Posts');
+      cfg.line('---');
+      cfg.line('');
+      cfg.line('> Add your own books or posts to our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Books-&-Posts)');
+      cfg.textFromFile('website/wiki/Books-&-Posts.md');
+    });
+
+    writeToFile('docs/videos.md', (cfg) => {
+      cfg.line('---');
+      cfg.line('id: videos');
+      cfg.line('title: Videos');
+      cfg.line('---');
+      cfg.line('');
+      cfg.line('> Add your own videos to our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Videos)');
+      cfg.textFromFile('website/wiki/Videos.md');
+    });
+  },
+
+  async publishSite(user = 'davertmik') {
+    // updates codecept.io website
+    await processChangelog();
+    copy('docker/README.md', 'docs/docker.md');
+    await this.wiki();
+    await chdir('website', async () => {
+      await exec('npm install');
+      await npmRun('publish-gh-pages', (cfg) => {
+        cfg.env('USE_SSH', 'true');
+        cfg.env('GIT_USER', process.env.GIT_USER || user);
+      });
+    });
+  },
+
+
+  async server() {
+    // run test server. Warning! PHP required!
+    await Promise.all([
+      exec('php -S 127.0.0.1:8000 -t test/data/app'),
+      npmRun('json-server'),
+    ]);
+  },
+
+  async release() {
+    // publish a new release on npm. Update version in package.json!
+    const packageInfo = JSON.parse(fs.readFileSync('package.json'));
+    const version = packageInfo.version;
+    await this.docs();
+    await this.publishSite();
+    await this.git((cmd) => {
+      cmd.tag(version);
+      cmd.push('origin master --tags');
+    });
+    await exec('npm publish');
+    console.log('-- RELEASED --');
+  },
+
+};
+
+async function processChangelog() {
+  const file = 'CHANGELOG.md';
+  let changelog = fs.readFileSync(file).toString();
+
+  // user
+  changelog = changelog.replace(/\s@([\w-]+)/mg, ' **[$1](https://github.com/$1)**');
+
+  // issue
+  changelog = changelog.replace(/#(\d+)/mg, '[#$1](https://github.com/Codeception/CodeceptJS/issues/$1)');
+
+  // helper
+  changelog = changelog.replace(/\s\[(\w+)\]\s/mg, ' **[$1]** ');
+
+  writeToFile('docs/changelog.md', (cfg) => {
```

Comment:
what is `cfg` for?

Reply:
this is the syntax in runio, with a callback function you configure the behavior of a task.
once I get more time I will write documentation for that